common/static/common/templates/components/paging-footer.underscore

```diff
@@ -1,20 +1,19 @@
-<nav class="pagination pagination-full bottom" aria-label="<%= paginationLabel %>">
-    <div class="nav-item previous"><button class="nav-link previous-page-link"><span class="icon fa fa-angle-left" aria-hidden="true"></span> <span class="nav-label"><%= gettext("Previous") %></span></button></div>
+<nav class="pagination pagination-full bottom" aria-label="<%- paginationLabel %>">
+    <div class="nav-item previous"><button class="nav-link previous-page-link"><span class="icon fa fa-angle-left" aria-hidden="true"></span> <span class="nav-label"><%- gettext("Previous") %></span></button></div>
     <div class="nav-item page">
         <div class="pagination-form">
-            <label class="page-number-label" for="page-number-input"><%= interpolate(
+            <label class="page-number-label" for="page-number-input"><%- StringUtils.interpolate(
                     gettext("Page number out of %(total_pages)s"),
-                    {total_pages: total_pages},
-                    true
+                    {total_pages: total_pages}
                 )%></label>
             <input id="page-number-input" class="page-number-input" name="page-number" type="text" size="4" autocomplete="off" aria-describedby="page-number-input-helper"/>
-            <span class="sr field-helper" id="page-number-input-helper"><%= gettext("Enter the page number you'd like to quickly navigate to.") %></span>
+            <span class="sr field-helper" id="page-number-input-helper"><%- gettext("Enter the page number you'd like to quickly navigate to.") %></span>
         </div>
 
-        <span class="current-page"><%= current_page %></span>
+        <span class="current-page"><%- current_page %></span>
         <span class="sr">&nbsp;out of&nbsp;</span>
```

Comment:
I think this is probably out of scope for your PR. This whole section needs to use sentences for the translators, as it isn't possible to translate snippets like this.